tensorflow/python/ops/math_ops.py

```diff
@@ -356,6 +356,27 @@ def _mul(x, y, name=None):
 @tf_export("math.subtract", "subtract")
 @dispatch.add_dispatch_support
 def subtract(x, y, name=None):
+  """Returns 'x' - 'y' element wise
+  
```

Comment:
Please remove all trailing whitespace (tabs and spaces at the end of the line).

You can configure your editor to strip these on save or at least to show them to you so they become visible